### ISSUE TITLE
Summer/Fall 2020 Release thread

### ISSUE THREAD

**Issue by ctschroeder** [CopticScriptorium/corpora#45]
Marcion (@amir-zeldes, @lancealanmartin)

- [x] Life of Pisenthius (@lancealanmartin)

    - [x]  doc 1: [Issue with repeating page numbers](https://github.com/CopticScriptorium/budge-dev/issues/5)

    - [x]  doc 2

    - [x] metadata issues in [this thread](https://github.com/CopticScriptorium/budge-dev/issues/6) must be resolved and addressed

    - [x] final metadata review by @ctschroeder 

- [x] Basilius on the End of the World

    - [x] metadata issues in [this thread](https://github.com/CopticScriptorium/budge-dev/issues/6) must be resolved and addressed

    - [x] final metadata review by @ctschroeder 

- [x] John of Constantinople (Chrys?)

    - [x]  Check authorship and circulation to determine corpus structure (@lancealanmartin will check and comm with (@ctschroeder) 

    - [x]  On Susannah

    - [x]  On repentance and Continence  (2 docs)

    - [x] metadata issues in [this thread](https://github.com/CopticScriptorium/budge-dev/issues/6) must be resolved and addressed

    - [x] final metadata review by @ctschroeder 

- [ ]  Eusebius On the Canaanite Woman (1 doc)

    - [x]  Check authorship and circulation to determine corpus structure (@lancealanmartin will check and comm with (@ctschroeder) 
    - [ ] metadata issues in [this thread](https://github.com/CopticScriptorium/budge-dev/issues/6) must be resolved and addressed

    - [x] final metadata review by @ctschroeder 

- [x] Pachomius Instructions (added by @ctschroeder 28 August)

    - [x] final metadata review by @ctschroeder 

AP @eplatte 

  - [x] 20-21 [everything complete on this thread](https://github.com/CopticScriptorium/ap-dev/issues/88)

Shenoute @bkrawiec & Dana Robinson

- [x] Dirt documents in XJ [everything complete on the Dirt section of this thread](https://github.com/CopticScriptorium/shenoute-discourses5-dev/issues/48)

- [x] Those documents [everything complete on the Those section of this thread](https://github.com/CopticScriptorium/shenoute-discourses5-dev/issues/48)

Entity Corpus (!!!!!)
- [x] (noun and proper noun entities identified and classified)

Pushed forward to Winter 2020/21

  - [ ] AP 43-45 [everything complete on this thread](https://github.com/CopticScriptorium/ap-dev/issues/87)

  - [ ] Besa/Shenoute from Goettingen

  - [ ] Johannes @ctschroeder

**Comment by ctschroeder** [comment]
version number will be 4.0. Version date will be automated.

**Comment by ctschroeder** [comment]
@amir-zeldes do we need to check with someone like Alin Suciu about whether the Marcion documents are really what they claim to be?  Or is there 100% certainty about author (vs stated author), title, whether it's a translation, etc.?

**Comment by amir-zeldes** [comment]
By all means, I'm very happy if Alin has time to take a look that would be great. I'm sadly not on expert on these aspects of ancient text attribution!

**Comment by ctschroeder** [comment]
These are not things he needs to look at. He knows the sources intimately and can quickly tell us the state of the field usually.

It’s important we figure out the latest assessment of authorship, etc., since people will be using our data for research. We don’t want our data to be unusable or misleading. So,  we don’t want a researcher publishing something about Athanasius in Coptic  translation when the data is not text authored by Athanasius, and this information is known in some sectors of the field. It’s not an issue of embarrassment about an error but producing a resource that leads to bad research. I will ask him about the texts in this list.

**Comment by amir-zeldes** [comment]
Yes, that's great. Like I said I'm all for it and happy to correct any errors that turn up.

**Comment by lancealanmartin** [comment]
Is there any update on Alin's comments on authorship? I found Greek versions of John of Constantinople (John IV or John the faster) and Eusebius "On the Caananite Woman" in Migne's PG. The later is actually written by John Chrysostom if Migne is correct, but his editions aren't infallible. I haven't found evidence contradicting the authorship of Basil for his discourse or John the presbyter for the Life of Pisentius but also nothing to confirm it.

**Comment by ctschroeder** [comment]
Working on it!  In the meantime you can checknPATHS-both the entries for the texts and authors and bibliography in those entries.

**Comment by lancealanmartin** [comment]
Great! The Paths entries are consistent with my note above, and, for now, I filled attributed_author and author fields in the metadata for all docs except for basilius.discourse. PATHs lists Basil as the stated author but does not list him or anyone as the work's creator.

**Comment by ctschroeder** [comment]
I added a new issue in the budge-dev repo to address metadata issues and linked to it above. More soon over on that thread.

**Comment by eplatte** [comment]
AP 20 and 21 are ready for review.

**Comment by ctschroeder** [comment]
The following Marcion corpora are **ready** for Monday; I told Alin we would publish on Monday. I made notes on anything I thought might be important for @amir-zeldes or @lancealanmartin to know:

- Pisentius ready except P 1 needs to be committed (crashing on commit)

- Basil (changed psbasilcaesarea in urn namespace to psbasil because the other Coptic Basil only has one text

- On Susanna: made changes to metadata (like adding JofC as attributed author as well as J Chrys) based on convo in https://github.com/CopticScriptorium/budge-dev/issues/6.  @amir-zeldes this document has an underscore in the name rather than a dot or dash -- do you care?

-On Penitence and Abstinence: changed author to John IV of Constantinople and title to On Penitence and Abstinence; if you want to change it back be sure to change all fields: document name at the top of the screen, author, URN, next/previous URNs msItem_title, title. @amir-zeldes penitence.01 needs to be committed -- crashed. UPDATE: was able to commit. (WEIRD)


Items that need @amir-zeldes attention
On the Canaanite Woman: changed doc name, corpus name at top; attributed author both Eus & J Chrys; author now says Pseudo-Chrysostom (my gut instinct - - we'll see what Alin says); corpus now pseudo.chrysostom; **corpus metadata has more names than doc metadata** -- which is correct?

Is there anything else from Budge? I noticed the Pachomius item was not on this list, so I added it. But it looks like it's a re-publication, not a new work.  Anything I am missing?

Also Lance, the info in the notes is SUPER. Thanks.

FYI the crashing on commit seems intermittent -- some docs commit if I try again later.

**Comment by lancealanmartin** [comment]
Fantastic! I think these are all the Budge docs for this release.

Right now, both On Susanna and On the Canaanite Woman are in the pseudo.chrysostom corpus. Mitchell and Liz worked on On Susanna.

**Comment by ctschroeder** [comment]
Ok. If the corpus metadata needs to be changed please go ahead and change it.

**Comment by ctschroeder** [comment]
Dirt is ready -- see comments in the shenoute thread

**Comment by amir-zeldes** [comment]
Fantastic - I've now added entities and fixed the 'hap' and other validation issues on docs assigned to me. Just waiting on:
  * Remaining docs of Those slated for release
  * AP20-21
  * decision on eusebius corpus/merger

I think that's it!

**Comment by ctschroeder** [comment]
AP 20-21  are done. I'll finish Those tonight. On the Canaanite woman -- send me a text before you release it. It should be good as pseudo.chrysostom, but I'm giving Alin til Monday if replies.

**Comment by ctschroeder** [comment]
Those is as good as it's going to get. Everything marked to-publish in GitDox is ready!

**Comment by amir-zeldes** [comment]
Fantastic, I'll get to work migrating them over the day (I'm teaching in between). I'll do Eusebius last, but I'm assuming it's merged into Chrysostom as discussed unless I hear back from you later this afternoon.

**Comment by amir-zeldes** [comment]
One question: is meta collection="unknown" correct for shenoute.those.GL269-270?

**Comment by ctschroeder** [comment]
Yes. Couldn’t figure it out. Either that or delete the field. 

Sent from my iPhone

**Comment by amir-zeldes** [comment]
unknown is fine, thanks!

**Comment by amir-zeldes** [comment]
All done!
